buffer/src/main/java/io/netty/buffer/AbstractByteBufAllocator.java

```diff
@@ -273,11 +274,7 @@ public int calculateNewCapacity(int minNewCapacity, int maxCapacity) {
         }
 
         // Not over threshold. Double up to 4 MiB, starting from 64.
-        int newCapacity = 64;
-        while (newCapacity < minNewCapacity) {
-            newCapacity <<= 1;
-        }
-
+        final int newCapacity = MathUtil.findNextPositivePowerOfTwo(Math.max(minNewCapacity, 64));
```

Comment:
The  comment above it is now a little inaccurate, since it doesn't always start from 64 or double up.

Perhaps something like "// Not over threshold. Use next power of 2 that is >=  minNewCapacity or 64."

Reply:
Good point, the final result should be the same: because doubling up starting from 64 means finding first next capacity of 2 over 64 and `minNewCapacity`